### PR TITLE
[feat] Rating and grid comp

### PR DESCRIPTION
**Pull request by rishichourasia**
Live Preview ----

Check these components Only - 

 Rating -  https://deploy-preview-13--scale-ui.netlify.app/components/rating/rating
 Grid -  https://deploy-preview-13--scale-ui.netlify.app/components/grid/grid

### CODE REVIEW

File: Components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.one {
+	background-color: rgb(73, 73, 73);
+	height: 100px;
+}
+.two {
+	background-color: #212121;
+	height: 100px;
+}
+.three {
+	background-color: rgb(211, 211, 211);
```
Comment (teja538) [inline Components/Grid/grid.css:37]: for colors use variables and also use either rgb or hex format, don't use mix of them

File: Components/Grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+				<div class="code-snippet">
+					<iframe
+						src="https://carbon.now.sh/embed?bg=rgba%28245%2C245%2C245%2C0%29&t=one-light&wt=none&l=htmlmixed&width=809.5000009536743&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=JetBrains+Mono&fs=15.5px&lh=144%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2509%2509%2509%2509%253Cdiv%2520class%253D%2522grid-2%2522%253E%250A%2509%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522%2522%253E%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522%2522%253E%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%253C%252Fdiv%253E"
+						style="
+							width: 810px;
+							height: 147px;
+							border: 0;
+							transform: scale(1);
+							overflow: hidden;
+						"
```
Comment (teja538) [inline Components/Grid/grid.html:124]: use separate class for iframe to define styles. so that you can reuse that class for all iframes for styling

**Review by teja538 — COMMENTED** [review]
suggestion : I observed in most of the stylings you are using px . try to use rem or % for defining margins, paddings and other sizes as they are relative units. everything else seems good

File: Components/Grid/grid.css (hunk 1 above)
Comment (rishichourasia) [reply Components/Grid/grid.css:37]: alright , thanks

File: Components/Rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	flex-direction: row;
+	justify-content: center;
+}
+
+/* component css  */
+.rating-star {
+	font-size: 1.5rem;
+}
+.fa-light {
+	color: grey;
```
Comment (AnuragSahu11) [inline Components/Rating/rating.css:28]: variable can be used

**Review by AnuragSahu11 — APPROVED** [review]
Teja has suggested good changes other than that code seems fine.

**Pull request merged**